build.gradle

```diff
@@ -157,25 +157,25 @@ subprojects {
             errorProneVersion = '2.4.0'
             errorProneJavacVersion = '9+181-r4173-1'
             findBugsJsr305Version = '3.0.2'
-            grpcVersion = '1.33.1'
-            guavaVersion = '30.0-android'
-            jacksonVersion = '2.11.3'
+            grpcVersion = '1.34.1'
+            guavaVersion = '30.1-android'
+            jacksonVersion = '2.12.0'
             jmhVersion = '1.26'
             junitVersion = '5.7.0'
-            mockitoVersion = '3.6.0'
+            mockitoVersion = '3.7.0'
```

Comment:
Same here: IMHO we should use a `+` for the patch version of test-only dependencies.

Reply:
This is not reproducible, I would encourage not to use `+` in any of the dependencies. It is very important to have reproducible builds and tests.

Reply:
I think there is a gradle feature for that https://docs.gradle.org/current/userguide/dependency_locking.html, but I haven't looked into it much yet.

Reply:
Agree that dynamic versions without locking isn't a good idea. And since we'd want to bump on minor versions anyways, I think bumping manually on patch versions isn't a huge deal and keeps the test dependencies consistent with non-test.